pkgs/development/python-modules/django-modeltranslation/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, django
+, pytestCheckHook
+, pytest-cov
+, pytest-django
+, parameterized
+}:
+let
+  # 0.18.12 was yanked from PyPI:
+  # <https://pypi.org/project/django-modeltranslation/0.18.12/>
+  version = "0.18.11";
+in
+buildPythonPackage {
+  pname = "django-modeltranslation";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "deschler";
+    repo = "django-modeltranslation";
+    rev = "v${version}";
+    hash = "sha256-WEtTy449z7Fo9+UmiM+QAuUJ5eQ1RFe1HrIqFrY3L9k=";
+  };
+
+  disabled = pythonOlder "3.6";
```

Comment:
```suggestion
```
No python version older than 3.11 is supported.

Reply:
I followed [this](https://django-modeltranslation.readthedocs.io/en/latest/installation.html#requirements) which says that `django-modeltranslation-0.18.*` supports Python 3.6 and higher, as long as Django has version 4.2 or higher. Could you explain your comment in more detail?

Reply:
but nix dosnt support versions lower than python 310 it is a line that is not nessecary

Reply:
Is it 3.10 or 3.11 then? Your messages seem to contradict each other...
I can't find any information about this, can you link to a definitive resource stating this?
Even if the line isn't strictly necessary, since the information is available it makes sense to have it in the derivation, right?